### PR TITLE
Add administrative regions in bragi

### PR DESCRIPTION
**Pull request by xlqian**
http://jira.canaltp.fr/browse/NAVP-592

add more admin details in street and address:

[x] tested with address  
[x] cannot test with street

```json
{
  "Autocomplete": {
    "type": "FeatureCollection",
    "geocoding": {
      "version": "0.1.0",
      "query": ""
    },
    "features": [
      {
        "type": "Feature",
        "geometry": {
          "coordinates": [
            2.3266459999999993,
            48.803326
          ],
          "type": "Point"
        },
        "properties": {
          "geocoding": {
            "id": "addr:2.326646;48.803326",
            "type": "house",
            "label": "22 Rue Guy de Gouyon du Verger, (Arcueil)",
            "name": "22 Rue Guy de Gouyon du Verger, (Arcueil)",
            "housenumber": "22",
            "street": "Rue Guy de Gouyon du Verger, (Arcueil)",
            "postcode": "94110",
            "city": "Arcueil",
            "administrative_regions": [
              {
                "id": "admin:fr:94003",
                "insee": "94003",
                "level": 8,
                "name": "Arcueil",
                "zip_code": "94110",
                "weight": 1,
                "coord": {
                  "lat": 48.80211180000001,
                  "lon": 2.3320691999999994
                }
              }
            ]
          }
        }
      }
    ]
  }
}
```

### CODE REVIEW

**Comment by xlqian** [comment]
I'm adding some unit tests...

File: libs/bragi/src/model.rs.in [hunk 1]
```diff
@@ -155,10 +157,10 @@ impl From<mimir::Addr> for GeocodingResponse {
         let label = Some(other.name);
         let housenumber = Some(other.house_number);
         let street_name = Some(other.street.name);
-        let (postcode, city) = other.street.administrative_region.map_or((None, None), |admin| {
-            (Some(admin.zip_code), Some(admin.name))
+        let (postcode, city, admins) = other.street.administrative_region.map_or((None, None, None), |admin| {
+            (Some(admin.zip_code.clone()), Some(admin.name.clone()), Some(vec![admin]))
         });
-
+
```
Comment (azime) [inline libs/bragi/src/model.rs.in]: space

File: libs/bragi/src/model.rs.in [hunk 2]
```diff
@@ -67,7 +67,7 @@ pub struct GeocodingResponse {
     //pub county: Option<String>,
     //pub state: Option<String>,
     //pub country: Option<String>,
-    //pub admin: Option<HashMap<String, String>>,
+    pub administrative_regions: Option<Vec<mimir::Admin>>,
```
Comment (antoine-de) [inline libs/bragi/src/model.rs.in]: we can use directly a `Vec<mimir::Admin>` no ?

Comment (xlqian) [inline libs/bragi/src/model.rs.in]: yep!

**Pull request merged**